openmc/examples.py

```diff
@@ -0,0 +1,634 @@
+import numpy as np
+
+import openmc
+import openmc.model
+
+
+def pwr_pin_cell():
+    """Create a PWR pin-cell model.
+
+    This model is a single fuel pin with 2.4 w/o enriched UO2 corresponding to a
+    beginning-of-cycle condition and borated water. The specifications are from
+    the `BEAVRS <http://crpg.mit.edu/research/beavrs>`_ benchmark. Note that the
+    number of particles/batches is initially set very low for testing purposes.
+
+    Returns
+    -------
+    model : openmc.model.Model
+        A PWR pin-cell model
+
+    """
+    model = openmc.model.Model()
+
+    # Define materials.
+    fuel = openmc.Material(name='Fuel')
+    fuel.set_density('g/cm3', 10.29769)
+    fuel.add_nuclide("U234", 4.4843e-6)
+    fuel.add_nuclide("U235", 5.5815e-4)
+    fuel.add_nuclide("U238", 2.2408e-2)
+    fuel.add_nuclide("O16", 4.5829e-2)
+
+    clad = openmc.Material(name='Cladding')
```

Comment:
Maybe it is just me, but I think the name "zircaloy" is more appropriate for a material, while "cladding" is a name more appropriate for a cell.